stripe/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -38,27 +38,30 @@ import kotlinx.coroutines.withContext
 import kotlinx.parcelize.Parcelize
 
 internal class DefaultFlowController internal constructor(
-    private val activity: ComponentActivity,
+    private val viewModelStoreOwner: ViewModelStoreOwner,
+    private val lifecycleScope: CoroutineScope,
+    private val activityLauncherFactory: ActivityLauncherFactory,
+    private val statusBarColor: () -> Int?,
+    private val authHostSupplier: () -> AuthActivityStarter.Host,
+    private val paymentOptionFactory: PaymentOptionFactory,
```

Comment:
Is this used?  It is not showing up in the diff.

Reply:
Yeah it's used further down in the class.